### PR TITLE
Display advisory names in siblings table

### PR DESCRIPTION
**Pull request by sarah256**

### CODE REVIEW

File: src/app/pipes/propertydisplay.ts [hunk 1]
```diff
@@ -34,7 +35,24 @@ export class PropertyValueDisplayPipe implements PipeTransform {
     const dtRegEx = /^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2})(?:\.\d+)?(?:Z|[-+]00(?::00)?)?$/;
     // Can't use typeof to determine if it's an Array
     if (value instanceof Array) {
-      return value.length.toString();
+      console.log(value);
+      if (value.length === 0) {
+        return '0';
+      } else {
+        if (value[0].advisory_name) {
+          let result = '';
```
Comment (lcarva) [inline src/app/pipes/propertydisplay.ts]: Could you just use `map` and `join` to achieve the same result?
```javascript
a = [{'advisory_name': 'bacon'}, {'advisory_name': 'spam'}, {'advisory_name': 'ham'}]
a.map(x => x.advisory_name).join(', ')
> "bacon, spam, ham"
```

File: src/app/pipes/propertydisplay.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { PipeTransform, Pipe } from '@angular/core';
 import { DatePipe } from '@angular/common';
+import { isEmpty } from 'rxjs/operators';
```
Comment (mprahl) [inline src/app/pipes/propertydisplay.ts]: Could you please remove this unneeded import? VSCode probably automatically imported this when you tab-completed something.

File: src/app/pipes/propertydisplay.ts [hunk 3]
```diff
@@ -34,7 +35,24 @@ export class PropertyValueDisplayPipe implements PipeTransform {
     const dtRegEx = /^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2})(?:\.\d+)?(?:Z|[-+]00(?::00)?)?$/;
     // Can't use typeof to determine if it's an Array
     if (value instanceof Array) {
-      return value.length.toString();
+      console.log(value);
```
Comment (mprahl) [inline src/app/pipes/propertydisplay.ts]: Could you please remove this? We don't need to log this.

File: src/app/pipes/propertydisplay.ts [hunk 4]
```diff
@@ -34,7 +35,24 @@ export class PropertyValueDisplayPipe implements PipeTransform {
...
+      if (value.length === 0) {
+        return '0';
+      } else {
+        if (value[0].advisory_name) {
```
Comment (mprahl) [inline src/app/pipes/propertydisplay.ts]: Adding a comment here explaining why we specifically check to see if this is an advisory would be nice.

**Pull request merged**